### PR TITLE
PEP8 cleanup

### PR DESCRIPTION
**Pull request by thezoggy**
Normally I wouldn't try to mess with import cleanup since the IDE is untrustworthy due to our less than optimal import practice (circular imports/having to do some to make freeze/py2exe happy).. but found two that are safe.

* articlecache

Status was added under:
https://github.com/sabnzbd/sabnzbd/commit/a8788da698dd73f9529ccf42e214f3955a36bfc7

No longer needed due to:
https://github.com/sabnzbd/sabnzbd/commit/12a622a21fbaf69dac4e554b40a08494a983048b

* bpsmeter

floor import was last touched under:
https://github.com/sabnzbd/sabnzbd/commit/2811e5ae7de0075039295965ebafab2cc564746d

floor was removed when you cleaned it up under:
https://github.com/sabnzbd/sabnzbd/commit/5c77f9d9b3d759120edc6b0cb652312096d9fcde
> not sure if this the reason of the rounding error/difference from glitter vs actual?

previous one had an issue, fixed and forced push.. but the pull request never updated only my repo.. not sure wtf is going on with that.. so just doing a new pull.
easier to review non-whitespace changes: https://github.com/sabnzbd/sabnzbd/pull/847/files?w=1

### CODE REVIEW

File: sabnzbd/nzbqueue.py [hunk 1]
```diff
@@ -441,6 +441,7 @@ def remove(self, nzo_id, add_to_history=True, save=True, cleanup=True, keep_basi
             nzo = self.__nzo_table.pop(nzo_id)
             nzo.deleted = True
             if cleanup and nzo.status not in (Status.COMPLETED, Status.FAILED):
+                # TODO: Use nzo.is_gone() ?
```
Comment (thezoggy) [inline sabnzbd/nzbqueue.py]: @Safihre pretty sure we should just use is_gone here, added note for ya to review later on..

File: sabnzbd/config.py [hunk 2]
```diff
@@ -93,6 +93,7 @@ def get_dict(self, safe=False):
 
     def set_dict(self, dict):
         """ Set value based on dictionary """
+        # TODO: Review if `dict` is actually correct or typo
```
Comment (Safihre) [inline sabnzbd/config.py]: Why do you think this is wrong? It seems to be a prototype to be overwritten by the inherited classes that use it to easily submit config settings.

File: sabnzbd/nzbqueue.py (hunk 1 above)
Comment (Safihre) [inline sabnzbd/nzbqueue.py]: Yes, indeed. You can change it to that.

File: sabnzbd/config.py (hunk 2 above)
Comment (thezoggy) [inline sabnzbd/config.py]: dict is a reserved word, and were passing an object to overwrite another object. so it just screams at me in my ide about it.. why i put a note to review :)

**Comment by thezoggy** [comment]
will make the changes and force push update here in a day..

**Comment by thezoggy** [comment]
rebased, updated, and force pushed.

**Comment by Safihre** [comment]
Merged! But removed all the `@UnresolvedImport` and `@IgnorePep8`.

**Comment by thezoggy** [comment]
@Safihre why not include them?

**Pull request closed without merging**